### ISSUE TITLE
calculate baseline metrics manually

### ISSUE THREAD

**Issue by simonpcouch** [simonpcouch/predictivebench#8]
Some of the baseline metrics don't look right to me. Possibly the literal uninformative metric rather than the null model?

Put together a deterministic pipeline that:

* Fetches the id and metric name from `modeling_dataset`
* Grabs the `train.csv` and `test.csv`
* Fit a null model to the training data
* Calculate the test set metric
* Subset in that metric value to `modeling_dataset$baseline`
